docs/components/modeler/web-modeler/milestones.md

```diff
@@ -1,50 +1,158 @@
+[deploy](/components/modeler/web-modeler/process-applications.md#deploy-and-run-a-process-application) them as a single bundle with just one click. This reduces the risk of having a broken deployment at runtime, and makes it more convenient to deploy related files.
+
+If you create a milestone on the main process of a process application, milestones are created for all other assets in the application, to make it easier to track or revert changes.
 
 :::note
 Milestones of resources belonging to a process application are tied to the main process and cannot be modified.
 :::
 
-### Restoring milestones
+## Compare milestones
+
+You can compare the change history between two milestones, either visually as a diagram or as code in an XML diff layout.
+
+1. Open the milestone history for your diagram.
+1. Ensure that the **Show changes** toggle in the **Changes** panel is turned on.
+1. Select the milestone that you want to compare. The previous milestone is automatically selected for comparison.
+
+:::note
+
+Turn off the **Show changes** toggle if you want to view individual milestones without comparison to the previous milestone.
+
+:::
+
+### Compare milestones in visual view
+
+To view BPMN diagram changes visually, select the **Visual view** tab.
+
+![milestones diffing](img/milestones/web-modeler-milestone-visual-diffing.png)
+
+- Differences between the milestones are highlighted visually on the diagram. For example, if an element was added, this change is highlighted in green with a plus symbol. Hover over a change to view more details.
+- Only differences that affect the execution of the BPMN process are highlighted.
+- The **Changes** panel shows the details of each change, including the type and identifier. Select a change to highlight it.
+
+:::note
+
+You can only use the **Code view** to compare changes in a DMN diagram. The **Visual view** only shows a view of the milestone.
+
+:::
+
+### Compare milestones in code view
 
-Hover over a milestone, click on the three vertical dots, and expand for more options.
-![milestones restore](img/milestones/web-modeler-milestone-restore.png)
-![milestones restore](img/milestones/web-modeler-milestone-restore-complete.png)
+To view BPMN and DMN diagram changes as code in an XML diff layout, select the **Code view** tab.
 
-### Comparing milestones
+![milestones diffing](img/milestones/web-modeler-milestone-code-diffing.png)
 
-Milestones can be compared visually. By enabling the diffing feature, the currently selected milestone is compared to its predecessor.
+- The XML for the previous milestone is shown on the left, with the currently selected milestone shown on the right.
+- Differences between the milestones are highlighted in the XML. For example, if an element was added, this change is highlighted in green.
+- Only differences that affect the execution of the BPMN process are highlighted.
```

Comment:
❌ This is not true. All changes between the two XMLs are shown.

Reply:
Change made